internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -157,6 +177,16 @@ $fluentd_config_dir = "$fluentd_base_dir\etc\td-agent"
 $fluentd_config_path = "$fluentd_config_dir\td-agent.conf"
 $fluentd_service_name = "fluentdwinsvc"
 
+$dotnet_tracing_msi_name = "signalfx-dotnet-tracing-0.2.9-x64.msi"
```

Comment:
Ideally the version number is a parameter/const.

Reply:
Same for bitness, there could be someone using Windows x86

Reply:
Interesting. It looks like this whole script is for "amd64" though, isn't it?

Reply:
ah, if we are limited to amd64 then... but we did had some users with 32bit versions of Windows in the past. On x64 for box the installer already installs both the x64 and x86 binaries.

Reply:
I think Windows Server 2008 was the last OS with 32bit. I really don't want to support it 😆

Reply:
I didn't know that @RassK !!!! 😮